src/content/docs/en/guides/images.mdx

```diff
@@ -28,7 +28,20 @@ Store your images in the `public/` folder if you want to avoid any processing or
 You can also choose to store your images remotely, in a [content management system (CMS)](/en/guides/cms/) or [digital asset management (DAM)](/en/guides/media/) platform.
 Astro can fetch your data remotely using APIs or display images from their full URL path.
 
- For extra protection when dealing with external sources, Astro's image components and helper function will only process (e.g. optimize, transform) images from [authorized image sources specified in your configuration](#authorizing-remote-images). Remote images from other sources will be displayed with no processing.
+For extra protection when dealing with external sources, Astro's image components and helper function will only process (e.g. optimize, transform) images from [authorized image sources specified in your configuration](#authorizing-remote-images). Remote images from other sources will be displayed with no processing.
+
+#### Asset Caching
```

Comment:
Might be interesting to note that local images are also cached.

Reply:
@Princesseuh Do you think maybe an entire section on Image Caching would make sense?

Reply:
I would say so! But it might be confusing for some users, since it's a bit more advanced than the rest and it's not something you interact with directly